tests/core/resource/test_filter.py

```diff
@@ -219,6 +219,34 @@ def test_exclude_returns_400_if_value_has_wrong_type(self):
         self.assertRaises(httpexceptions.HTTPBadRequest,
                           self.resource.collection_get)
 
+    def test_contains_with_strings(self):
+        self.validated['querystring'] = {'contains_colors': ["red", "blue"]}
+        result = self.resource.collection_get()
+        values = [item['colors'] for item in result['data']]
+        for value in values:
+            assert "red" in value and "blue" in value
+
+    def test_contains_with_integer(self):
+        self.validated['querystring'] = {'contains_fib': [3]}
+        result = self.resource.collection_get()
+        values = [item['fib'] for item in result['data']]
+        for value in values:
+            assert 3 in value
+
+    def test_contains_any_with_strings(self):
+        self.validated['querystring'] = {'contains_colors': ["red", "blue"]}
+        result = self.resource.collection_get()
+        values = [item['colors'] for item in result['data']]
+        for value in values:
+            assert 'red' in value or 'blue' in value
+
+    def test_contains_any_with_integer(self):
+        self.validated['querystring'] = {'contains_fib': [3, 5]}
+        result = self.resource.collection_get()
+        values = [item['fib'] for item in result['data']]
+        for value in values:
+            assert 3 in value or 5 in value
```

Comment:
These tests are inaccurately named.

Reply:
Whould `test_contains_with_string_select_records_with_color` be better? Any suggestion?